src/components/ContributionsEpic.tsx

```diff
@@ -160,17 +164,20 @@ export const ContributionsEpic: React.FC<Props> = ({
     tracking,
     countryCode,
     numArticles,
+    onReminderOpen,
 }: Props) => {
+    const [isReminderActive, setIsReminderActive] = useState(false);
+
     const { heading, backgroundImageUrl, showReminderFields } = variant;
 
     return (
-        <section className={wrapperStyles} data-target="contributions-epic">
+        <section className={wrapperStyles}>
             {backgroundImageUrl && (
                 <div className={imageWrapperStyles}>
                     <img
                         src={backgroundImageUrl}
                         className={imageStyles}
-                        alt="Image for Guardian contributions message"
+                        alt="Guardian contributions message"
```

Comment:
Is this purely visual? If so it can be an empty `alt=""` otherwise it should be more descriptive of the content of the image.

Reply:
This is a dynamic image added in the Epic tool, the API doesn't expose any meta. Probably better to use `alt=""` as you suggest.

Reply:
```suggestion
                        alt=""
```